edx_sga/constants.py

```diff
@@ -1,6 +1,6 @@
 """Constants"""
 
-BLOCK_SIZE = 8 * 1024
+BLOCK_SIZE = 2**10 * 8  # 8kb
```

Comment:
Are you just trying to make this easier to read?

Reply:
yes, this was old value in system,